### PR TITLE
Update docs for renamed uaa credentials broker.

### PR DESCRIPTION
**Pull request by jmcarp**
Goes with https://github.com/cloudfoundry-community/deployer-account-broker/pull/19. Let's make sure the proposed names make sense before merging either.

### CODE REVIEW

**Comment by brittag** [comment]
I added a commit with a few minor edits for clarity. Do we need an explanation of how the "service account" service was previously called a "deployer account" service? Do we need to tell people using the old `deployer-account` plan to do anything to transition to the new one? (Is this a breaking change?)

**Comment by jmcarp** [comment]
Existing accounts should continue to work. Hopefully the alias for the deployer account url will help if users get confused.

**Comment by brittag** [comment]
Cool, this makes sense to me then! I'll approve it but leave it to somebody else to merge since I made the last commit.

**Review by brittag — APPROVED** [review]

**Comment by afeld** [comment]
Is it available in both environments?

File: content/docs/apps/continuous-deployment.md [hunk 1]
```diff
@@ -28,16 +28,16 @@ You can provision a deployer account with permission to deploy to a single space
 1. Create a deployer account service instance
 
     ```bash
-    $ cf create-service deployer-account deployer-account my-deployer-account
+    $ cf create-service cloud-gov-service-account space-deployer my-service-account
```
Comment (mogul) [inline content/docs/apps/continuous-deployment.md:31]: I'd make it `my-deployer-account` to encourage people to choose names relevant to them for specific instance that they create.

File: content/docs/services/cloud-gov-identity-provider.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+Plan Name | Description | Price
+--------- | ----------- | -----
+`oauth-client` | Register a UAA client for app authentication | Free
+
+## How to create an instance
+
+To create a service instance, run the following command:
+
+```bash
+cf create-service cloud-gov-identity-provider oauth-client my-uaa-client \
```
Comment (mogul) [inline content/docs/services/cloud-gov-identity-provider.md:22]: How about `my-oauth-client`... remember, they don't know anything about UAA.

File: content/docs/services/cloud-gov-service-account.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    parent: services
+title: cloud.gov service account
+name: "cloud-gov-service-account"
+description: "Manage cloud.gov service accounts with access to your organization"
+status: "Beta"
+aliases:
+- /docs/services/deployer-account/
+---
+
+To set up your application to be deployed with an automated system, you need a deployer account that has access to push your application.
```
Comment (mogul) [inline content/docs/services/cloud-gov-service-account.md:13]: "need" is a little strong. I'd say something like "you should use an account that has access to push your application that's not associated with a specific user (to insulate against people leaving your organization, changing projects, etc.)"

File: content/docs/services/cloud-gov-service-account.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+## Plans
+
+Plan Name | Description | Price
+--------- | ----------- | -----
+`space-deployer` | Provision one account with "SpaceDeveloper" access to the space where the service is created | Free
```
Comment (mogul) [inline content/docs/services/cloud-gov-service-account.md:19]: s/one/an/

**Review by mogul — DISMISSED** [review]
Only small changes requested, and also the answer to @afeld's question!

**Comment by jmcarp** [comment]
@afeld: both services are only available in govcloud.

@mogul: some of the changes you mentioned would revert changes that @brittag made on this PR. Can I get you two to sort these out? 😄

**Review by sharms — APPROVED** [review]

**Pull request merged**